spring-integration-core/src/main/java/org/springframework/integration/graph/CompositeMessageHandlerNode.java

```diff
@@ -33,7 +35,7 @@ public class CompositeMessageHandlerNode extends MessageHandlerNode {
 
 	private final List<InnerHandler> handlers = new ArrayList<>();
 
-	public CompositeMessageHandlerNode(int nodeId, String name, MessageHandler handler, String input, String output,
+	public CompositeMessageHandlerNode(int nodeId, String name, MessageHandler handler, @Nullable String input, @Nullable String output,
```

Comment:
Please, revise: the `inputChannel` cannot be null for any `IntegrationConsumer`.